packages/terra-list/docs/guides/SingleSelectList.md

```diff
@@ -0,0 +1,108 @@
+# Terra List - Implementing a Single Select List
+
+In previous versions of the terra-list a single select style list could be created as either a controlled version of SelectableList or uncontrolled version of SingleSelectList.  These implementations suffered from inflexibility and performance concerns. Going foward terra-list is more granular, though this puts more reponsibility on the consumer to properly update their list items with the appropriate state. The following is a guide to addressing those concerns in your implementation.
```

Comment:
A few typos in this paragraph. I noticed responsibility and forward

Reply:
Updated.